packages/ag-charts-community/src/chart/toolbar/toolbar.ts

```diff
@@ -538,6 +540,22 @@ export class Toolbar extends BaseModuleInstance implements ModuleInstance {
         return button;
     }
 
+    private updateToolbarAriaLabel(group: ToolbarGroup, alignElement?: HTMLElement) {
+        if (!alignElement) {
+            const align = this[group].align ?? 'start';
+            const position = this[group].position ?? 'top';
+            alignElement = this.positionAlignments[position][align];
+            if (!alignElement) return;
+        }
+        const map = {
+            annotations: 'ariaLabelAnnotationsToolbar',
+            annotationOptions: 'ariaLabelAnnotationOptionsToolbar',
+            ranges: 'ariaLabelRangesToolbar',
+            zoom: 'ariaLabelZoomToolbar',
+        } as const;
+        alignElement.ariaLabel = this.ctx.localeManager.t(map[group]);
```

Comment:
I'm happy to go with this for now. But we could put it in the `AgToolbarGroup` options and `ToolbarGroupProperties` and access it here as `this[group].ariaLabel` if we want users to be able to change it?

Reply:
Also, a `.ag-charts-toolbar__align` is not limited to just one group of buttons (annotations/zoom etc).

If a user sets the same position and alignment for two button groups it will put them next to each other in this same element. And so will set the aria-label to be the latest updated.

The other possibility is the `.ag-charts-toolbar__section` element, but a group can be split across multiple sections, e.g. the annotations has the delete button in its own section:

<img width="328" alt="Screenshot 2024-06-26 at 09 41 50" src="https://github.com/ag-grid/ag-charts/assets/3817697/da0a3c41-c98c-4889-a221-48c20f7c3973">

But that would end up with multiple elements having the same aria-label.

I think the current solution is fine, as by default all the groups are separate, but its worth noting.

Reply:
The `this[group].ariaLabel` sounds like a good idea, but the user can already change this value simply by updating the localised string. So we can keep it as it for now.

The solution for multiple toolbars in a one `.ag-charts-toolbar__align` will require a bit more. I think that we'll need an intermediate container between `.ag-charts-toolbar__align` and `.ag-charts-toolbar__section`. Let's do this in a follow up PR.